src/NServiceBus.AcceptanceTesting/ScenarioContext.cs

```diff
@@ -28,10 +29,17 @@ public void AddTrace(string trace)
 
         ConcurrentQueue<string> traceQueue = new ConcurrentQueue<string>();
 
+        internal Dictionary<string, LogLevel> LogLevels = new Dictionary<string, LogLevel>();
+
+        public void SetLogLevel(string logger, LogLevel level)
```

Comment:
I think that would make sense. On the `ScenarioContext.Log` level you can always filter based on log level using LINQ.